pyproject.toml

```diff
@@ -2,7 +2,7 @@
 name = "nautobot"
 # Primary package version gets set here. This is used for publishing, and once
 # installed, `nautobot.__version__` will have this version number.
-version = "1.1.0"
+version = "1.1.1-beta.1"
```

Comment:
Need to update this of course. :-)

Reply:
Yeah! I'll be fixing these all in one shot once #776 is merged!